helm-chart/renku/values.yaml

```diff
@@ -131,8 +131,8 @@ keycloak:
   ## keycloak admin username/password is supplied.
   initRealm:
     image:
-      repository: renku/init-realm
-      tag: 'latest'
+      repository: degano/init-realm
```

Comment:
I think this should be pushed to the renku repo as well

Reply:
I will `chartpress --reset` before merging.